### ISSUE TITLE
Both shifting and scaling operations for Range instance is not documented

### ISSUE THREAD

**Issue by titsuki** [perl6/doc#818]
Here is the example of clone-with-op method

```
$ perl6 -e 'my $range = 0..3; my $new-range = $range.clone-with-op(&[+] , 10); say $new-range;'
10..13
```

See also
https://github.com/rakudo/rakudo/blob/nom/src/core/Range.pm#L595-L600

**Comment by pmichaud** [comment]
Is "clone-with-op" part of the language spec?  I notice there aren't any tests for it in roast, which means it's not really a part of "Perl 6".  It feels to me more like a "Rakudo internals" method which the general user shouldn't be using.

Pm

**Comment by titsuki** [comment]
@pmichaud 
Thanks for your reply.
I've changed the title of this issue according to the following test cases.
https://github.com/perl6/roast/blob/master/S02-types/range.t#L292-L325

**Comment by gfldex** [comment]
ENOSPEC, closed

**Comment by AlexDaniel** [comment]
Reopening this because #819 was reopened. Can be closed again if we decide so in #835.

**Comment by titsuki** [comment]
Fixed by #1797

**Comment by zoffixznet** [comment]
> Fixed by #1797

Seeing how the ticket has "NOTSPECCED" label... is this feature now specced?
